### DISCUSSION TITLE
Dropdown when using the TAB key get focus trap

### DISCUSSION THREAD

**Discussion by dsgnow** [primefaces/community#241] [PrimeNG] [answered]
The p-dropdown traps focus when I navigate to it using the TAB key, while other inputs work correctly. Is this a BUG, or do I need to pass some flag to make it work in the new version? Im using latest 16.5.0.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We're currently implementing accessibility to input components and we've just completed the dropdown component. Keyboard actions are changed, the mentioned issue is solved and the solution will be available with the upcoming release this Wednesday.

**Comment by dsgnow** [comment]
Unfortunately, the problem still persists. I currently have version 16.6.0. Are work on this still ongoing?

**Reply by cetincakiroglu** [reply]
We're planning to release it next week.

**Reply by ThoSap** [reply]
@dsgnow PrimeNG 16.7.0 is now released https://github.com/primefaces/primeng/releases/tag/16.7.0

**Reply by dsgnow** [reply]
In the latest version (16.7.1), tabbing to the next dropdown now works, but it also automatically changes the value in that dropdown. For example, if nothing is selected in the dropdown and I press the Tab key to move to the next element, the first item in the dropdown changes to that value. You can also observe this behavior on the main PrimeNG website. https://primeng.org/dropdown -> when you enter the page, don't click on the dropdown, just press the Tab key; if the dropdown has the "touched" flag, it works correctly.

**Comment by dsgnow** [comment]
Will the correction for this bug be implemented in the next version?
